src/AutoRest.CSharp/Mgmt/Models/MgmtClientOperation.cs

```diff
@@ -134,14 +134,23 @@ public MethodSignature MethodSignature
         private FormattableString BuildDescription()
         {
             var pathInformation = _operations.Select(operation =>
-                (FormattableString)$@"<item>
+            {
+                string resourceTypeItem = "";
+                if (operation.Resource != null)
+                    resourceTypeItem = $@"
+<item>
+<term>Resource Type</term>
+<description>{operation.Resource.ResourceType}</description>
+</item>";
```

Comment:
do we need to add something to indicate that this operation does not correspond to a resource type so that it does not support api-version overriding?

Reply:
I think it's better to add the comment to the method of overriding api-version about how to figure out the ResourceType or whether some methods are supported/unsupported. The ResourceType listed with the method is more like general information about the method like the OperationId and RequestPath. They are there to help user get them easily and correctly no matter who want to use them, so it's better for the methods that use them to determine/comment how they are expected to be used or supported.